src/QuadForm/indefiniteLLL.jl

```diff
@@ -0,0 +1,357 @@
+
+################################################################################
+#
+#                       indefinite LLL reduction
+#
+################################################################################
+
+################################################################################
+#                  Helpful
+################################################################################
+
+#=
+  _round_matrix(M::MatElem) -> MatElem
+
+Returns the matrix $M$ where all entries are replaced by their rounded values.
+
+$Example$
+==========
+julia> _round_matrix(QQ[1//3 -2 3//2; 5//3 0 -6; -7//3 -81//50 0])
+[0 -2 2]
+[2 0 -6] 
+[-2 -2 0]
+=#
+function _round_matrix(M::MatElem)
```

Comment:
This function is used only once and it is a basic function. Usually it is also always better to keep the `map_entries` as they are in the code so that any person reading what you have done can skip this shortcut.